examples/text-generation/textgen_pipeline/pipeline.py

```diff
@@ -0,0 +1,56 @@
+import os
+import sys
+
+import torch
+from transformers import TextGenerationPipeline
+
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.pardir)))
+try:
+    from utils import initialize_model
+except:
+    raise
```

Comment:
This try-except block was initially added to prevent ruff from complaining. That said, this has now been removed as users can run the pipeline by updating `PYTHONPATH`